pandas/core/indexes/base.py

```diff
@@ -122,6 +122,24 @@ class Index(IndexOpsMixin, PandasObject):
     Notes
     -----
     An Index instance can **only** contain hashable objects
+
+    Examples
+    --------
+    >>> pd.Index([1, 2, 3])
+    Int64Index([1, 2, 3], dtype='int64')
+
+    >>> pd.Index(list('abc'))
+    Index(['a', 'b', 'c'], dtype='object')
+
+    See Also
+    ---------
+    RangeIndex : Index implementing a monotonic integer range (very
+                 memory-efficient)
+    CategoricalIndex : Index with :class:`Categorical` s.
+    MultiIndex : A multi-level, or hierarchical, Index
+    IntervalIndex : an Index of :class:`Interval` s.
```

Comment:
Did you check locally that those links work inside a 'See also' description ?

Reply:
I hadn't. I've started building the docs and will to the check.

Reply:
``PeriodIndex`` has no link, others are ok.